javalib/src/main/scala/java/net/StandardSocketOptions.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2022 Arman Bilge
+ *
+ * Original code is from the armanbilge/epollcat project at
+ * https://github.com/armanbilge/epollcat/
+ *
+ * For full original license notice , see:
+ *     https://github.com/scala-native/scala-native/blob/main/LICENSE.md
+ *
+ * Additional code provided by the Scala Native project carries the
+ * Scala Native license, described in the same LICENSE.md.
+ */
+
+package java.net
+
+object StandardSocketOptions {
+
+  val IP_MULTICAST_IF: SocketOption[java.net.NetworkInterface] =
+    new StdSocketOption("IP_MULTICAST_IF", classOf)
+
+  val IP_MULTICAST_LOOP: SocketOption[java.lang.Boolean] =
+    new StdSocketOption("IP_MULTICAST_LOOP", classOf)
+
+  val IP_MULTICAST_TTL: SocketOption[java.lang.Integer] =
+    new StdSocketOption("IP_MULTICAST_TTL", classOf)
```

Comment:
I'm unable to find a bridge method for this option defined in posixlib. Perhaps we can add it, if you don't mind? :)

Reply:
@armanbilge It dawned on me whilst I was driving back from town today that you `IP_MULTICAST_TTL` question
was one of those trick interview questions, and that I had fallen for it.

`IP_MULTICAST_TTL` only applies to UDP sockets, known in Java as the Datagram class.  The Socket class and
it clade are all stream sockets. There used to be a Datagram option for Socket, but that was deprecated in something like Java 1.1 or .2.     

Scala Native has not implement the Datagram related classes.  It has been on my wish list for years, but
other things have been more important. Sorry.  Implementing them it is almost certainly  more than a week's work.
A good reason for using a Datagram in the first place is to do multicast, so an extensive set of Tests would
need to be written.

I have done a lot of work over the past while implementing and exercising UDP in posixlib.
What are you trying to accomplish?  Perhaps we could talk off line and I might be able
to guide and/or misdirect you. 

I do not want to stray too far from the main objective of this PR.

Reply:
I created Issue #2818 to capture this discussion so that it does not get lost when
this PR gets merged.   Arman, I hope I described this discussion well, feel free
to correct in that issue.

One way of implementing takes hours; helping the 1000 cats dream together takes
more time & life-energy. For me it is worth spending those to live in a community with
some harmony.